packages/labs/gen-wrapper-react/src/index.ts

```diff
@@ -149,45 +151,60 @@ const tsconfigTemplate = () => {
   );
 };
 
+const getTypeImports = (declarations: LitElementDeclaration[]) => {
+  // We only need type imports for events.
+  const refs = declarations.flatMap(({events}) =>
+    Array.from(events.values()).flatMap((e) => e.type?.references ?? [])
+  );
+  return getImportsStringForReferences(refs);
+};
+
+const getElementTypeExportsFromImports = (imports: string) =>
+  imports.replace(/(?:^import|(\s)*import)/gm, '$1export type');
```

Comment:
Why the `|(\s)*import` and `$1` replacement? Don't grok what could precede the import. Add comment?

Long-term, I think we'll want to add a `getExportsStringForReferences` function, since in the general case it's not always that you'll want to import the same references that you export...

Reply:
Fixed and added comment. Was unsure of format of source string but it's `import ...\nimport...` which is fine.